### DISCUSSION TITLE
remove /home cooldowns

### DISCUSSION THREAD

**Discussion by Fasted9** [NillerMedDild/Enigmatica6#1929] [Q&A]
Is there any way to remove the /home, /warp, or /back cooldowns? I am running this on a server from a separate laptop.

**Comment by MuteTiefling** [comment]
We use the issue tracker exclusively for bug reports, development and feature requests.  However, this issue appears to be a support request. Please use the [support channel on our Discord](https://discord.gg/kZ7RdrD) to get help.
